R/as_rtf.R

```diff
@@ -112,3 +124,122 @@ as_rtf.gsBinomialExactTable <- function(
 
   invisible(x_out)
 }
+#' @rdname as_rtf
+#' @param title Title of the report.
+#' @param footnote_p_onesided Footnote for one-side p-value. 
+#' @param footnote_appx_effect_at_bound Footnote for approximate effect treatment at bound.
+#' @param footnote_p_cross_hr1 Footnote for cumulative type I error.
+#' @param footnote_p_cross_hr0.5 Footnote for cumulative power under the alternate hypothesis.
+#' @param footnote_specify Vector of string to put footnote super script.
+#' @param footnote_text Vector of string of footnote text.
+#' @importFrom r2rtf rtf_title rtf_footnote
+#' @export
```

Comment:
Can we have some examples there? Please make sure there will be one example cover the implementation of `footnote_specify`.